python/external_config/external_config_loader.py

```diff
@@ -166,7 +166,7 @@ def request_configurations(self, project_id):
         # in rapid succession.
         #
         # note: not using a generator since we are deleting in the loop
-        for task_id in self._task_ids.keys():
+        for task_id in list(self._task_ids.keys()):
```

Comment:
Any chance we can add a test for this function? We clearly didn't have coverage and this is a vital part of browser integration.

Reply:
You'll likely mock things a bit so it doesn't spawn a subprocess.